server/internal/mv/toolset.go

```diff
@@ -457,9 +459,15 @@ func DescribeToolset(
 		}
 	}
 
+	orgMetadata, err := orgRepo.GetOrganizationMetadata(ctx, toolset.OrganizationID)
+	if err != nil {
+		return nil, oops.E(oops.CodeUnexpected, err, "failed to get organization metadata").Log(ctx, logger)
+	}
+
 	return &types.Toolset{
 		ID:                     toolset.ID.String(),
 		OrganizationID:         toolset.OrganizationID,
+		AccountType:            orgMetadata.GramAccountType,
```

Comment:
Why add this here instead of just hitting a `describeOrg` where needed?

Reply:
Also DescribeOrg could make an outbound call to Polar. Which we dont want in this path.